skiko/src/commonMain/kotlin/org/jetbrains/skia/BreakIterator.kt

```diff
@@ -305,14 +305,6 @@ class BreakIterator internal constructor(ptr: NativePointer) : Managed(ptr, _Fin
         _text?.close()
     }
 
-    /**
-     * Create a copy of this iterator
-     */
-    fun clone(): BreakIterator {
-        Stats.onNativeCall()
-        return BreakIterator(withErrorGuard("Failed to clone") { _nClone(_ptr, it) })
-    }
-
```

Comment:
I reckon this change would probably require bumping Skiko version to 0.9 🤔

Reply:
Okay, we'll build the next version as 0.9.0

Reply:
Explanation why it is deleted in an internal channel: https://jetbrains.slack.com/archives/C02DDNREC77/p1738678000232729

- `breakIteratorCloneTest` is failing with an access violation in Debug mode with additional checks, so using it wasn't safe
- it is not used anywhery

Reply:
I split this change off into a separate PR: #1024